src/pages/alpha/TodoList/components/Todo.jsx

```diff
@@ -0,0 +1,73 @@
+import clsx from 'clsx';
+import { Input } from './Input';
+
+export function Todo({ list, toggleEdit, toggleStatus, onUpdate, onDelete, onChange }) {
+  return (
+    <ul className="in-progress mt-6 flex flex-col space-y-12" onChangeCapture={onChange}>
+      {list.map((item) => {
+        return (
+          item.status === 'todo' && (
+            <li
+              className="flex items-center justify-between rounded-lg bg-zinc-50 p-4 "
+              key={item.id}
+            >
+              <div className="flex items-center justify-center space-x-4">
+                <button
+                  className={clsx(
+                    'flex h-6 w-6 items-center justify-center',
+                    'rounded-full border  transition-all duration-300 active:scale-90',
+                    item.status === 'complete'
+                      ? 'border-green-400 text-green-400'
+                      : 'border-gray-400 text-gray-400'
+                  )}
+                  onClick={() => toggleStatus(item)}
+                >
+                  ✓
+                </button>
+                <label className={clsx('text-gray-600', item.state === 'edit' ? 'hidden' : '')}>
+                  {item.title}
+                </label>
+                <Input
+                  name={item.id}
+                  value={item.title}
+                  className={clsx(item.state === 'edit' ? '' : 'hidden')}
```

Comment:
https://www.robinwieruch.de/conditional-rendering-react/
JSX 可以寫 js 語法，這種叫 conditional rendering
跟 vue 的 `v-if` 和 `v-else` 很像

```suggestion
                {item.state !== 'edit' && (
                  <label className={clsx('text-gray-600')}>
                    {item.title}
                  </label>
                )}
                {item.state === 'edit' && (
                  <Input
                    name={item.id}
                    value={item.title}
                  />
                )}
```

Reply:
另外 `clsx` 的用法也不是這樣...
只能說這樣也可以 work

Reply:
好喔～！